config/vufind/config.ini

```diff
@@ -1323,6 +1323,8 @@ gl          = "Galician"
 [LanguageSettings]
 ; Comma-separated list of languages to display in right-to-left mode
 rtl_langs = "ar,he"
+; Comma-separated list of fallback locales to use for missing translations
+fallbacks = "nl-be:nl,*:en"
```

Comment:
As language fallbacks (parent language relations) will probably not differ between text domains I suggest we drop the discussed `@fallback` directive (`@parent_language`)  and instead opt for specifiying these globally by configuration.

This could be done as above. But we could also introduce an additional configuration file (i.e. `i18n.ini`) where fallbacks are then specified within their own section:
```
[Fallbacks]
nl-be = nl
*     = en
```

Reply:
Yes, this makes sense to me. I think the setting is okay as you have currently implemented it -- there is definitely something to be said for an `i18n.ini`, but if we are going to go that route, I think we should move all the i18n settings for the sake of consistency; that's a more disruptive change that we can consider as a subsequent steps. This approach is fine for now.